polaris-react/src/components/AlphaStack/AlphaStack.scss

```diff
@@ -7,7 +7,7 @@
   --pc-stack-gap-lg: var(--pc-stack-gap-md);
   --pc-stack-gap-xl: var(--pc-stack-gap-lg);
   display: flex;
-  flex-direction: column;
+  flex-direction: var(--pc-stack-order);
```

Comment:
we need to set `flex-direction: column-reverser` for the `withinContentContainer` style below so adding order `AlphaStack`

Reply:
I would break out the update to `AlphaStack` and create a separate PR against `main`. That way `reverseOrder` is immediately available on `AlphaStack` in case others need to use it. Otherwise the change will be stuck on this branch which isn't going to get merged in for a while yet.

Reply:
broke this out into #7842